src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/task/OpprettOppgaveForOpprettetBehandlingTask.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.ef.sak.behandlingsflyt.task
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.ef.sak.oppgave.OppgaveService
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.internal.TaskService
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.util.Properties
+import java.util.UUID
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = OpprettOppgaveTask.TYPE,
+    beskrivelse = "Opprett oppgave i GOSYS for opprettet revurdering",
+    maxAntallFeil = 3
+)
+class OpprettOppgaveForOpprettetBehandlingTask(
+    private val oppgaveService: OppgaveService,
+    private val taskService: TaskService
+) : AsyncTaskStep {
+
+    data class OpprettOppgaveTaskData(
+        val behandlingId: UUID,
+        val saksbehandler: String,
+        val beskrivelse: String? = null,
+        val hendelseTidspunkt: LocalDateTime = LocalDateTime.now(),
+        val mappeId: Long? = null
+    )
+
+    override fun doTask(task: Task) {
+        val data = objectMapper.readValue<OpprettOppgaveTaskData>(task.payload)
+        val oppgaveId = oppgaveService.opprettOppgave(
+            behandlingId = data.behandlingId,
+            oppgavetype = Oppgavetype.BehandleSak,
+            tilordnetNavIdent = data.saksbehandler,
```

Comment:
Hvordan blir dette for automatisk journalføring? Blir da tilordnetNavIdent = "VL" ? Det ønsker vi kanskje ikke?

Reply:
Håndtert nå med 
`if (data.saksbehandler == SikkerhetContext.SYSTEM_FORKORTELSE) null else data.saksbehandler`